snuba/clickhouse/optimize/optimize.py

```diff
@@ -157,7 +170,7 @@ def run_optimize_cron_job(
     return len(partitions_to_optimize)
 
 
-def get_partitions_to_optimize(
+def get_partitions_from_clickhouse(
```

Comment:
`OptimizedPartitionTracker` has a method on it called `get_partitions_to_optimize`, and I thought it was confusing that the two (quite different) methods had the same name

Reply:
Can you add some documentation as to what this function is actually supposed to do? It's fairly long and it makes some design decisions, but it's not obvious at a glance for what purpose

Reply:
added a doc comment, let me know if you think it makes sense